Makefile

```diff
@@ -633,6 +633,17 @@ endif
 	git commit -m "Adding $(MLRUN_VERSION) tag contents" --allow-empty; \
 	git push origin $$BRANCH_NAME
 
+.PHONY: api-backward-compatibility
+api-backward-compatibility: ## Check for api backward compatibility breakage
+ifndef BASE_PATH
+	$(error BASE_PATH is undefined)
```

Comment:
See that everything is prefixed with `MLRUN_`, there is a reason for that, it's environment variables, and you don't want one project's env vars to intersect with other project's env vars, therefore we prefix everything.
Since this env var value is very specific and I don't expect we'll ever use it for anything other, I think the name should be very specific as well, I would name this `MLRUN_BC_TESTS_BASE_CODE_PATH`